keps/sig-node/3673-kubelet-parallel-image-pull-limit/README.md

```diff
@@ -345,9 +346,15 @@ This can inform certain test coverage improvements that we want to do before
 extending the production code to implement this enhancement.
 -->
 
-New unit test will be added to image_manager_test.go.
+New unit test is be added to image_manager_test.go.
```

Comment:
typo?

Reply:
good catch. I was meant to say that new unit test is added already. Updated.